<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SymbolMangerClass</class>
 <widget class="QWidget" name="SymbolMangerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SymbolManger</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>简单符号</string>
      </attribute>
      <widget class="QLabel" name="ctrlSymbolPreview">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>151</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>填充颜色：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>边框颜色：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>边框宽度：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_fillcolor">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_strokecolor">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ctrlStrokeWidth">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>230</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.300000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="ctrlConfirm">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>340</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ctrlCancel">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>340</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>边框样式：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="ctrlPenStyle">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>270</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>隐藏线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>实线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>短横虚线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>点虚线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>交替虚线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>双点交替虚线</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>SVG符号</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeView" name="ctrlTreeView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>281</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="ctrlScrollWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>349</width>
            <height>410</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTableView" name="tableView">
             <property name="autoScrollMargin">
              <number>16</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>getSelectStrokeWidth()</slot>
 </slots>
</ui>
